### PR TITLE
Add command guide for UG

### PR DESCRIPTION
**Pull request by jhqiu21**
* find person
* find event
* view person
* view event
* filter person
* filter event

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -109,23 +109,115 @@ Examples:
...
+  e.g. `Hair Hiking` will return `Hair Cut`, `Park Hiking`, `Hiking`
+
+Examples:
+* `find event Hiking` returns `Hiking` and `Park Hiking`
+* `find event Hair Oscars` returns `Hair Cut`, `Oscars`<br>
+  ![result for 'find hair oscars'](images/findHairOscarsResult.png)
+
+### Viewing person by name: `view person`
+
+Views the details of a specific person whose name exactly matches the given keywords.
```
Comment (fusin02) [inline docs/UserGuide.md:150]: i think we should emphasise that view is special in showing DETAILED details (address and email address), which makes it different from the rest. Maybe we could say: "Views additional detailed information of a specific person..... which includes Email address and address" sth like that maybe

Comment (jhqiu21) [reply docs/UserGuide.md]: Good tips! We need to emphasize that! Fixed already.

**Review by fusin02 — APPROVED** [review]
lgtm!

File: docs/UserGuide.md (hunk 1 above)
Comment (fusin02) [reply docs/UserGuide.md]: Thank you !

**Comment by Haris-Irfan** [comment]
LGTM!

**Pull request merged**